tox.ini

```diff
@@ -67,7 +67,7 @@ basepython =
     pypy: pypy
 
 commands =
-    py.test {env:TESTPATH} {posargs}
+    py.test --tb=short {env:TESTPATH} {posargs}
```

Comment:
Instead of copying this option everywhere, i'd suggest to add a `pytest.ini`:

```
[pytest]
addopts = --tb=short
```

Or alternatively in `setup.cfg`:

```
[tool:pytest]
addopts = --tb=short
```